### PR TITLE
https://issues.redhat.com/browse/ACM-5887 KI MSA status 2.8

### PR DESCRIPTION
**Pull request by oafischer**
2.8 and 2.7

### CODE REVIEW

File: release_notes/known_issues_continuity.adoc [hunk 1]
```diff
@@ -220,3 +220,14 @@ oc delete appliedmanifestwork <the-left-appliedmanifestwork-name>
 
 //[#known-issues-volsync]
 //== Volsync known issues
+
+[#msa-status-unknown]
+=== The _managed-seriveaccount_ add-on status shows _Unknown_
```
Comment (swopebe) [inline release_notes/known_issues_continuity.adoc]: 'service'

File: release_notes/known_issues_continuity.adoc [hunk 2]
```diff
@@ -220,3 +220,14 @@ oc delete appliedmanifestwork <the-left-appliedmanifestwork-name>
...
+//2.8:ACM-5887
+
+The managed cluster `appliedmanifestwork` `addon-managed-serviceaccount-deploy` is removed from the imported managed cluster if you are using the Managed Service Account without enabling it on the :mce: resource of the new hub cluster.
+
```
Comment (swopebe) [inline release_notes/known_issues_continuity.adoc:226]: {mce}

File: release_notes/known_issues_continuity.adoc [hunk 3]
```diff
@@ -220,3 +220,14 @@ oc delete appliedmanifestwork <the-left-appliedmanifestwork-name>
...
+The managed cluster is still imported to the new hub cluster, but 
+the `managed-serviceaccount` add-on status shows `Unknown`.
+ 
+You can recover the `managed-serviceaccount` add-on after enabling the Managed Service Account in the :mce-short: resource. See link:../business_continuity/backup_restore/backup_msa.adoc#enabling-auto-import[Enabling automatic import] to learn how to enable the Managed Service Account.
```
Comment (swopebe) [inline release_notes/known_issues_continuity.adoc]: {mce-short}

File: release_notes/known_issues_continuity.adoc [hunk 4]
```diff
@@ -218,5 +218,16 @@ You can also delete the remaining `appliedmanifestwork` manually by running the
...
```
Comment (swopebe) [inline release_notes/known_issues_continuity.adoc]: {mce-short}

**Review by swopebe — CHANGES_REQUESTED** [review]

**Review by swopebe — APPROVED** [review]

**Review by sahare — APPROVED** [review]
Looks good to me, Thanks!

**Pull request merged**